bbc/simorgh
# Add Whatsapp promo to Indonesian articles, update Japanese nav

Resolves JIRA n/a

Overall changes
======
- Update the in article podcast promo to promote BBC Indonesia's Whatsapp channel
- Replace the Gaza war topic with a Middle East crisis topic on the Japanese nav

Code changes
======

- Updated the podcastPromo section of src/app/lib/config/services/indonesia.ts to promote Whatsapp 
- Edited Navigation section of src/app/lib/config/services/japanese.ts to update the second link 


Testing
======
1. Open a BBC Indonesia article, the in-article podcast promo should promote whatsapp and open their whatsapp channel
2. Open the Japanese Front page, the second link in the nav should be 中東危機 and point to https://www.bbc.com/japanese/topics/cn7y603vz3lt


<img width="339" alt="indo_promo" src="https://github.com/user-attachments/assets/a2900227-057c-443c-881f-047f427d8990">
<img width="392" alt="japanese_nav" src="https://github.com/user-attachments/assets/df352a33-0fdc-4ae8-8904-e4c5ae40983e">





Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
